.github/workflows/gcp-test-deploy.yml

```diff
@@ -0,0 +1,315 @@
+        with:
+          short-length: 7
+
+      - name: Downcase network name for disks
+        run: |
+          NETWORK_CAPS=${{ inputs.network }}
+          echo "NETWORK=${NETWORK_CAPS,,}" >> $GITHUB_ENV
+
+      # Setup gcloud CLI
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v0.7.1
+        with:
+          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
+          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
+          token_format: 'access_token'
+
+      - name: Create GCP compute instance
+        id: create-instance
+        if: (${{ inputs.needs_constants }} && steps.changed-files-specific.outputs.any_changed == 'true' && github.event_name == 'push') || ${{ inputs.${{ inputs.test_id }}s }} || !${{ inputs.needs_constants }}
+        run: |
+          gcloud compute instances create-with-container "${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}" \
+          --boot-disk-size 100GB \
+          --boot-disk-type pd-ssd \
+          --create-disk name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}",device-name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}",size=100GB,type=pd-ssd \
+          --container-mount-disk mount-path="${{ inputs.zebra_state_path }}",name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}" \
+          --container-image debian:buster \
+          --container-restart-policy=never \
+          --machine-type ${{ env.MACHINE_TYPE }} \
+          --scopes cloud-platform \
+          --metadata=google-monitoring-enabled=true,google-logging-enabled=true \
+          --tags ${{ inputs.app_name }} \
+          --zone "${{ env.ZONE }}"
+          sleep 30
+
+      - name: ${{ inputs.test_name }}
+        id: ${{ inputs.test_id }}
+        if: ${{ steps.create-instance.outcome == 'success' }}
+        run: |
+          gcloud compute ssh \
+          ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command \
+          "docker run -e ${{ inputs.test_variable }}=1 -t --name ${{ inputs.test_id }} \
+          --mount type=bind,source=/mnt/disks/gce-contaißners-mounts/gce-persistent-disks/${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},target=${{ inputs.zebra_state_path }} \
+          ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
+
+          EXIT_CODE=$(\
+          gcloud compute ssh \
+          ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command="docker wait ${{ inputs.test_id }}")
+
+          exit ${EXIT_CODE}
+
+      - name: Get state version from constants.rs
+        run: |
+          STATE_VERSION=""
```

Comment:
Minor: `STATE_VERSION appears unused`

```suggestion
```

Reply:
Fixed by: [fix(rw): minor fixes](https://github.com/ZcashFoundation/zebra/pull/4271/commits/044e944715ff59bccd0eb2f415f9a4737cfb336a)

Reply:
This change is not in that commit, please re-apply.

Reply:
It's being used here https://github.com/ZcashFoundation/zebra/pull/4271/files#diff-b62c23498eced92972377fdfbaf08c0d1518cabd782645cca9bfc2bfa12711edR144

Reply:
It's a bit confusing, but:
- `LOCAL_STATE_VERSION` is used in this step
- `STATE_VERSION` is created in `$GITHUB_ENV` for the next step, but not actually used in this step

You could rename `LOCAL_STATE_VERSION` to `STATE_VERSION`, that would fix this issue.

Reply:
I'm just going to apply this fix, because it seems to have been missed twice.